acceptance/acceptance_test.go

```diff
@@ -109,10 +110,10 @@ func testAccept(t *testing.T, InprocessMode bool, singleTest string) int {
 	cloudEnv := os.Getenv("CLOUD_ENV")
 
 	if cloudEnv == "" {
-		server := StartServer(t)
-		AddHandlers(server)
+		defaultServer := StartServer(t)
+		AddHandlers(defaultServer)
 		// Redirect API access to local server:
-		t.Setenv("DATABRICKS_HOST", server.URL)
+		t.Setenv("DATABRICKS_HOST", defaultServer.URL)
```

Comment:
nit: can you keep this name unchanged? I don't see 'server' being used later for anything.

Reply:
It's more descriptive to call it `defaultServer` since overrides spawn a new server. Can change though if you feel strongly about this.